esp/esdllib/esdl_script.cpp

```diff
@@ -1024,7 +1541,7 @@ class CEsdlCustomTransform : public CInterfaceOf<IEsdlCustomTransform>
         else
             prefixes.appendUniq("");
     }
-    virtual void toDBGLog() override
+    virtual void toDBGLogx() override
```

Comment:
Is the trailing x in the function name a typo?

Reply:
oops that was me tracking down usage.  I'll change the name back.